lib/src/dependency.dart

```diff
@@ -15,3 +19,13 @@ final _firebaseAuthProvider = Provider<FirebaseAuth>((ref) {
 final _googleSignInProvider = Provider<GoogleSignIn?>((ref) {
   return GoogleSignIn();
```

Comment:
use arrow function everywhere you can in this file:

`final _firebaseAuthProvider = Provider<FirebaseAuth>((ref) => FirebaseAuth.instance);`